test/unit/pages/plans/[id].test.js

```diff
@@ -106,4 +106,29 @@ describe('PlanSummary', () => {
       expect(container.querySelector('.legal-text')).not.toBeInTheDocument();
     });
   });
+
+  describe('adding an action', () => {
+    it('does not render an action if goal does not exist', () => {
+      const plan = {
+        id: '1',
+        firstName: 'Mr',
+        lastName: 'Don'
+      };
+      const { queryByText } = render(<PlanSummary plan={plan} />);
+
+      expect(queryByText('Our Actions')).toBeNull();
```

Comment:
I think it might be more readable as expect(getByText('Our Actions')).not.toBeInTheDocument();

Reply:
It can't getByText, because that text doesn't exist, so it just times out I think